templates/explore/code.tmpl

```diff
@@ -3,7 +3,6 @@
 	{{template "explore/navbar" .}}
 	<div class="ui container">
 		<form class="ui form ignore-dirty" style="max-width: 100%">
-			<input type="hidden" name="tab" value="{{$.TabName}}">
```

Comment:
The fact that it was explicitly sent with the form perplexes me:
Why would it be added if it was unneeded?
Or is it **now** unneeded?

Reply:
yes, it's not used in explore page anymore, just some leftover after some other refactoring